packages/transport-webtransport/README.md

```diff
@@ -12,11 +12,11 @@ A [libp2p transport](https://docs.libp2p.io/concepts/transports/overview/) based
 ## Example
 
 ```js
```

Comment:
I think since #2250 if we change the code fence type to `TypeScript` it'll get checked during CI which should have picked this up:

```suggestion
```TypeScript
```